### PR TITLE
Documentation-Fix RST formatting Typos

### PR DESCRIPTION
**Pull request by zm711**
Was looking at docs and noticed a couple places where the rst wasn't formatted. Just fixed.

### CODE REVIEW

**Review by alejoe91 — APPROVED** [review]
Thanks!

File: doc/development/development.rst [hunk 1]
```diff
@@ -164,7 +164,7 @@ As described in the `black documentation <https://black.readthedocs.io/en/stable
 
 The following are some styling conventions that we follow in SpikeInterface:
 
-#. Avoid using abreviations in variable names (e.g., use :code:`recording` instead of :code:`rec`). It is specially important to avoid single letter variables.
+#. Avoid using abreviations in variable names (e.g., use :code:`recording` instead of :code:`rec`). It is especially important to avoid single letter variables.
```
Comment (h-mayorquin) [inline doc/development/development.rst:167]: This was on me. I was not aware of that distinction. Thanks!

Comment (zm711) [reply doc/development/development.rst:167]: @h-mayorquin In American English they are actually used pretty interchangeable, but the unofficial rule is specially is used in spoken and especially in written English. It's just convention though.

Comment (h-mayorquin) [reply doc/development/development.rst:167]: Thanks. Useful to know.

**Comment by h-mayorquin** [comment]
I have a generla question @zm711 , what ares of the documentation do you think are lacking in terms of content? What do you tihnk would be (or would have been) helpful to add?

**Comment by zm711** [comment]
@h-mayorquin That's an interesting question. I think the beginner section for install could actually be moved to the docs. Currently it links to the Github folder and if someone is a beginner the jump to all the files and markdown might already be a scary leap and they are starting with python (or moving from Matlab where the install is all behind the scenes/toolkits). The number of times someone (myself included when I started) installed a scientific program in base instead of an environment is unfortunate. So I would like to have at least conda instructions (since most scientists start with conda/minconda) for creating a virtual environment in the beginner docs.

I think the API docs needs to be updated (ie we need to keep adding docstrings). Since most IDEs will include docstring info it is actually really helpful to know if a function consumes a `Sorting` or `WaveformExtractor`. And if the docstring is unclear a beginner would go to the documentation only to see that the same lack of info is present in the API section. (More advanced would just go to source to fill in the gaps).

Based on the questions/issues. I think an explanation of lazy and what parts of spikeinterface are lazy and when someone should save is important. I know Alessio is adding a piece to the documentation, but scientists not coming from any computer background might not really know disk vs RAM and the benefits of using one or the other. So a fully fleshed out section on some of these "computer science" tricks that si uses behind the scenes might help newbies understand someone of the errors since currently it is hard to know when to switch. 

I think a final lower priority section needs to explain the ProbeInterface and Neo portions. I think someone of the Neo terminology makes sense once you are used to it, but an explanation might be more beneficial. Currently there is just a link to the Neo page. But if any Neo api is being exposed through the si api then documentation will make it less likely for beginners to make mistakes. Same for ProbeInterface, some recording systems have info and some don't and I think one fundamental issue when starting spike sorting is the realization that the numbers on the screen aren't the same as the numbers on the actual electrode due to wiring. The ProbeInterface documentation covers that fact, but I think a small section here explaining that as well would be beneficial. More like an FYI: did you know that without a wiring diagram you might not be sorting what you think? Then a brief explanation what is meant by wiring and then link to ProbeInterface for deeper info.

Which leads to just more examples. The example has one mega example (+Neuropixel example). So having some more troubleshooting examples of each module. There are spiketutorials, but making that clearer in the documentation or including them in the docs might be helpful. Some people don't read the docs they just go to juypter and try to run the examples, but then they use a different recording equipment or they want to use a different sorter and now it doesn't work. So they hop to another example hoping to fill in the pieces. It's just a different learning style, so having a richer example section might help the "learn by doing" people.

Just my two cents though. Hopefully something out of that is helpful.

**Comment by h-mayorquin** [comment]
This is SUPER USEFUL.

I will open and issue  or discussion and expand upon this with some ideas that I have.

Thanks a bunch for taking the time to write @zm711

**Comment by samuelgarcia** [comment]
@zm711 merci beaucoup.
We should organize doc sprint soon.

**Pull request merged**